### ISSUE TITLE
vpp has orphaned libraries

### ISSUE THREAD

**Issue by gtkramer** [clearlinux/distribution#1592]
Orphaned libraries in the vpp-plugins subpackage:
```
#     18957    496 -rwxr-xr-x   1 root     root       506216 Dec 10 18:42 /usr/lib/libvppinfra.so.19.08.1
#     18948     44 -rwxr-xr-x   1 root     root        42952 Dec 10 18:42 /usr/lib/libvapiclient.so.19.08.1
#     18946    116 -rwxr-xr-x   1 root     root       116728 Dec 10 18:42 /usr/lib/libsvm.so.19.08.1
#     18950     96 -rwxr-xr-x   1 root     root        96472 Dec 10 18:42 /usr/lib/libvcl_ldpreload.so.19.08.1
#     18956    176 -rwxr-xr-x   1 root     root       178688 Dec 10 18:42 /usr/lib/libvppcom.so.19.08.1
#     18953    108 -rwxr-xr-x   1 root     root       109240 Dec 10 18:42 /usr/lib/libvlibmemoryclient.so.19.08.1
#     18954  10908 -rwxr-xr-x   1 root     root     11169544 Dec 10 18:42 /usr/lib/libvnet.so.19.08.1
#     18947     36 -rwxr-xr-x   1 root     root        34712 Dec 10 18:42 /usr/lib/libsvmdb.so.19.08.1
#     18952    192 -rwxr-xr-x   1 root     root       192784 Dec 10 18:42 /usr/lib/libvlibmemory.so.19.08.1
#     18949     16 -rwxr-xr-x   1 root     root        14240 Dec 10 18:42 /usr/lib/libvatplugin.so.19.08.1
#     18951    828 -rwxr-xr-x   1 root     root       845608 Dec 10 18:42 /usr/lib/libvlib.so.19.08.1
#     18955     40 -rwxr-xr-x   1 root     root        38816 Dec 10 18:42 /usr/lib/libvppapiclient.so.19.08.1
```

**Comment by thiagomacieira** [comment]
What does "orphaned libraries" mean?

**Comment by fenrus75** [comment]
they're in /usr/lib/ not /usr/lib64/

**Comment by ahkok** [comment]
I left these in on purpose - upstream needs to fix these paths and the build system is not cooperating.

**Comment by phmccarty** [comment]
This issue was fixed in March 2021 with the update from 19.08.1 to 21.01:

https://github.com/clearlinux-pkgs/vpp/commit/3167b372af1a82128823206ae75d30605dfde0b3
